packages/fxa-auth-server/lib/payments/google-play/subscriptions.ts

```diff
@@ -48,9 +55,13 @@ export class PlaySubscriptions
     if (Container.has(PlayBilling)) {
       this.playBilling = Container.get(PlayBilling);
     }
+
+    if (Container.has(StripeHelper)) {
+      this.stripeHelper = Container.get(StripeHelper);
```

Comment:
Just echoing Ben here from another comment I saw: StripeHelper should exist here since subscriptions.enabled is true at this point.

Reply:
Yeah I pushed up a change and replied to Ben [here](https://github.com/mozilla/fxa/pull/10964#discussion_r751481217), but it broke some other tests due to `PlaySubscriptions` being instantiated as part of setting up the `mozillaSubscriptionRoutes`.

I can try and unset it or something before each of those tests.